docs/source/manual/openfpga_shell/openfpga_commands/setup_commands.rst

```diff
@@ -521,3 +521,26 @@ write_fabric_pin_physical_location
   .. option:: --verbose
 
     Show verbose log
+
+.. _openfpga_setup_commands_report_reference:
+
+report_reference
+~~~~~~~~~~~~~~~~~~~~
+
+  Write reference information of each child module under a given parent module to a YAML file
```

Comment:
Also update documentation about the YAML file format of this command. See example: https://github.com/lnis-uofu/OpenFPGA/blob/master/docs/source/manual/file_formats/fabric_hierarchy_file.rst